# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perl/cross-lm.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- perl-5.8.8/Makefile.SH.vanilla	2006-03-21 12:58:12.000000000 +0100
+++ perl-5.8.8/Makefile.SH	2006-03-21 12:58:20.000000000 +0100
@@ -248,7 +248,7 @@
 ext = \$(dynamic_ext) \$(static_ext) \$(nonxs_ext)
 DYNALOADER = lib/auto/DynaLoader/DynaLoader\$(LIB_EXT)
 
-libs = $perllibs $cryptlib
+libs = $perllibs $cryptlib -lm
 
 public = perl\$(EXE_EXT) $suidperl utilities translators
 
